src/domains/sign/io_sign_user_func.tf

```diff
@@ -0,0 +1,93 @@
+locals {
+  io_sign_user_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME                        = "node"
+      WEBSITE_NODE_DEFAULT_VERSION                    = "16.13.0"
+      WEBSITE_VNET_ROUTE_ALL                          = "1"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
+      FUNCTIONS_WORKER_PROCESS_COUNT                  = 4
+      AzureWebJobsDisableHomepage                     = "true"
+      NODE_ENV                                        = "production"
+      CosmosDbConnectionString                        = module.cosmosdb_account.connection_strings[0]
+      CosmosDbDatabaseName                            = module.cosmosdb_sql_database_user.name
+      StorageAccountConnectionString                  = module.io_sign_storage.primary_connection_string
+      userUploadedBlobContainerName                   = azurerm_storage_container.uploaded_documents.name
+      userValidatedBlobContainerName                  = azurerm_storage_container.validated_documents.name
+      WaitingMessageQueueName                         = azurerm_storage_queue.waiting_message.name
+      IoServicesApiBasePath                           = "https://api.io.italia.it"
```

Comment:
https://api.io.pagopa.it

Reply:
it's the same architecture only change are more restrictive cipher suites... anyway we can migrate on 16th December :)